baselines/ppo2/ppo2.py

```diff
@@ -5,7 +5,7 @@
 from collections import deque
 from baselines.common import explained_variance, set_global_seeds
 from baselines.common.policies import build_policy
-from overcooked_ai_py.agents.agent import AgentPair, ToMModel
+from human_ai_robustness.human_ai_robustness_utils import eval_and_viz_tom
```

Comment:
We shouldn't have imports from specific project repos in the main file, or it will break it if people from other projects try to use it (which will not have your project installed). Should maybe keep a flag somewhere (in the kwargs dictionary is probably the simplest way to do this) so that you can import this function only if you are running ppo2.py from your project.

Reply:
I've made eval_and_viz_tom a method of ToMModel now, so we don't need to import from human_ai_robustness

Reply:
Yep, that is even more elegant